app/cothorityd/cothorityd.go

```diff
@@ -96,3 +108,10 @@ func runServer(ctx *cli.Context) {
 
 	server.RunServer(config)
 }
+
+// checkConfig contacts all servers and verifies if it receives a valid
+// signature from each.
+func checkConfig(c *cli.Context) error {
```

Comment:
It's not for now but later we should find a way to integrate directly this command in "one-line" that includes the `cli.Command` and the given handler.